### PR TITLE
Reverted extra changes back

### PR DESCRIPTION
**Pull request by aklochkova**
Fixes https://github.com/OHDSI/Atlas/issues/1338

this PR is only for fixing 'master' branch
it doesn't include sql scripts for unique checking and permissions fixes

### CODE REVIEW

**Comment by aklochkova** [comment]
@pavgra hi! because that code changes were my own initiative and they aren't relative to this task. and there is no code style convention, so i decided to revert all of them back.

File: src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java [hunk 1]
```diff
@@ -183,18 +221,22 @@ public GenerateSqlRequest() {
     return getSourceJdbcTemplate(source).query(psr.getSql(), psr.getSetter(), summaryMapper);
   }
 
-  private final RowMapper<AnalysisReport.StrataStatistic> strataRuleStatisticMapper = (rs, rowNum) -> {
-    AnalysisReport.StrataStatistic statistic = new AnalysisReport.StrataStatistic();
+  private final RowMapper<AnalysisReport.StrataStatistic> strataRuleStatisticMapper = new RowMapper<AnalysisReport.StrataStatistic>() {
```
Comment (pavgra) [inline src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java]: Why was lambda removed?

File: src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java [hunk 2]
```diff
@@ -216,13 +258,17 @@ private String formatBitMask(Long n, int size) {
...
   }
 
-  private final RowMapper<StratifyReportItem> stratifyResultsMapper = (rs, rowNum) -> {
-    StratifyReportItem resultItem = new StratifyReportItem();
-    resultItem.bits = rs.getLong("strata_mask");
-    resultItem.totalPersons = rs.getLong("person_count");
-    resultItem.timeAtRisk = rs.getLong("time_at_risk");
-    resultItem.cases = rs.getLong("cases");
-    return resultItem;
+  private final RowMapper<StratifyReportItem> stratifyResultsMapper = new RowMapper<StratifyReportItem>() {
```
Comment (pavgra) [inline src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java]: Why was lambda removed?

File: src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java [hunk 3]
```diff
@@ -277,14 +323,37 @@ private String getStrataTreemapData(int analysisId, int targetId, int outcomeId,
     return treemapData.toString();
   }
 
+  public IRAnalysisDTO analysisToDTO(IncidenceRateAnalysis  analysis) {
```
Comment (pavgra) [inline src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java]: Seems like converter

File: src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java [hunk 4]
```diff
@@ -277,14 +323,37 @@ private String getStrataTreemapData(int analysisId, int targetId, int outcomeId,
...
   @Override
-  public List<IRAnalysisShortDTO> getIRAnalysisList() {
+  public List<IRAnalysisService.IRAnalysisListItem> getIRAnalysisList() {
 
     return getTransactionTemplate().execute(transactionStatus -> {
       Iterable<IncidenceRateAnalysis> analysisList = this.irAnalysisRepository.findAll();
-      return StreamSupport.stream(analysisList.spliterator(), false)
-              .map(analysis -> conversionService.convert(analysis, IRAnalysisShortDTO.class))
-              .collect(Collectors.toList());
+      return StreamSupport.stream(analysisList.spliterator(), false).map(p -> {
```
Comment (pavgra) [inline src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java]: Why was converter removed?

File: src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java [hunk 5]
```diff
@@ -303,20 +372,20 @@ public IRAnalysisDTO createAnalysis(IRAnalysisDTO analysis) {
     // If there's a way to get the Entity into the persistence manager so findOne() returns this newly created entity
     // then we could create the entity here (without persist) and then call saveAnalysis within the same Tx.
     IncidenceRateAnalysis newAnalysis = new IncidenceRateAnalysis();
-    newAnalysis.setName(analysis.getName())
-            .setDescription(analysis.getDescription());
+    newAnalysis.setName(analysis.name)
```
Comment (pavgra) [inline src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java]: Please use getters

File: src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java [hunk 6]
```diff
@@ -325,7 +394,7 @@ public IRAnalysisDTO getAnalysis(final int id) {
     return getTransactionTemplate().execute(transactionStatus -> {
       IncidenceRateAnalysis a = this.irAnalysisRepository.findOne(id);
       ExceptionUtils.throwNotFoundExceptionIfNull(a, String.format(NO_INCIDENCE_RATE_ANALYSIS_MESSAGE, id));
-      return conversionService.convert(a, IRAnalysisDTO.class);
+      return analysisToDTO(a);
```
Comment (pavgra) [inline src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java]: Please use converter

File: src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java [hunk 7]
```diff
@@ -470,10 +539,19 @@ public AnalysisReport getAnalysisReport(final int id, final String sourceKey, fi
 
     Source source = this.getSourceRepository().findBySourceKey(sourceKey);
 
-    AnalysisReport.Summary summary = IterableUtils.find(getAnalysisSummaryList(id, source), summary12 -> ((summary12.targetId == targetId) && (summary12.outcomeId == outcomeId)));
+    AnalysisReport.Summary summary = IterableUtils.find(getAnalysisSummaryList(id, source), new Predicate<AnalysisReport.Summary>() {
```
Comment (pavgra) [inline src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java]: Why was lambda removed?

File: src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java [hunk 8]
```diff
@@ -470,10 +539,19 @@ public AnalysisReport getAnalysisReport(final int id, final String sourceKey, fi
...
+      @Override
+      public boolean evaluate(AnalysisReport.Summary summary) {
+        return ((summary.targetId == targetId) && (summary.outcomeId == outcomeId));
+      }
+    });
 
-    Collection<AnalysisReport.StrataStatistic> strataStats = CollectionUtils.select(getStrataStatistics(id, source), 
-            summary1 -> ((summary1.targetId == targetId) && (summary1.outcomeId == outcomeId)));
+    Collection<AnalysisReport.StrataStatistic> strataStats = CollectionUtils.select(getStrataStatistics(id, source), new Predicate<AnalysisReport.StrataStatistic>() {
```
Comment (pavgra) [inline src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java]: Why was lambda removed?

File: src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java [hunk 9]
```diff
@@ -500,9 +578,11 @@ public GenerateSqlResult generateSql(GenerateSqlRequest request) {
   @Override
   public IRAnalysisDTO copy(final int id) {
     IRAnalysisDTO analysis = getAnalysis(id);
-    analysis.setId(null); // clear the ID
-    analysis.setName(getNameForCopy(analysis.getName()));
-    return createAnalysis(analysis);
+    analysis.id = null; // clear the ID
```
Comment (pavgra) [inline src/main/java/org/ohdsi/webapi/service/IRAnalysisService.java]: Please use setters

**Pull request closed without merging**